src/rars/venus/editors/jeditsyntax/SyntaxDocument.java

```diff
@@ -1,12 +1,3 @@
-/*
- * SyntaxDocument.java - Document that can be tokenized
- * Copyright (C) 1999 Slava Pestov
- *
- * You may use and modify this package for any purpose. Redistribution is
- * permitted, in both source and binary form, provided that this notice
- * remains intact in all source distributions of this package.
```

Comment:
This part needs to be amended to the License.txt file to have attribution since it isn't the same as the other deleted headers.

Reply:
amended how, I put it under what is written??